docs/content/modeling/roles-and-permissions.mdx

```diff
@@ -43,20 +43,19 @@ For example:
 
 There are advantages to implementing roles and permissions within <ProductName format={ProductNameFormat.ShortForm}/>, such as:
 
-- Breaking down existing roles to have more fine grained permissions. This allows your application to check whether a _user_ has access to a certain object without having to explicitly check that specific _users_ role.
-- Introduce new roles/permissions or consolidate roles without affecting your application behavior. For example: if in your app all the checks are for the fine permissions `check('bob', 'booking_adder', 'trip:Europe')` instead of `check('bob', 'owner', 'trip:Europe')`, and then you later decide `owners` can no longer add bookings to a `trip`, you can remove the relation within the `trip` type with no code changes in your application, and all the permissions will automatically honor the change.
+- Breaking down existing roles to have more fine grained permission, whucg allows your application to check whether a user has access to a certain object without having to explicitly check that specific users role.
+- Introduce new roles/permissions or consolidate roles without affecting your application behavior. For example: if your app's checks are for the fine permissions `check('bob', 'booking_adder', 'trip:Europe')` instead of `check('bob', 'owner', 'trip:Europe')`, and you later decide `owners` can no longer add bookings to a `trip`, you can remove the relation within the `trip` type with no code changes in your application, and all the permissions will automatically honor the change.
 
```

Comment:
Consider breaking this out from an UL. It's quite detailed and somewhat code-heavy.